### DISCUSSION TITLE
Local Dev Environment Error (Database Outline does not exist)

### DISCUSSION THREAD

**Discussion by tawandamoyo** [outline/outline#3961] [General] [answered]
![db](https://user-images.githubusercontent.com/17948595/184312148-fc16600e-122c-4cdd-8343-3fadad1b3518.png)
I am trying to set up a local dev environment, on Linux. When I run `make up` I get the following error: 

`ERROR: database "outline" does not exist`

I was under the impression that the db would be created during setup but clearly I am mistaken. Would appreciate any help.

**Comment by tommoor** [comment, marked as answer]
Try `yarn db:reset`

**Reply by tawandamoyo** [reply]
Thanks, It worked
